src/Adapters/FHIRPatientAdapter.php

```diff
@@ -452,6 +465,22 @@ public function interfaceToModel( PatientInterface $patient )
         $fhirPatient->addTelecom( $email );
 
 
+        $address = new FHIRAddress();
+        $line = new FHIRString();
+        $line->setValue($patient->getStreet());
```

Comment:
@Leo24 No, The iPhone UI has two lines, address-line-1 and address-line-2. We are taking those two lines and storing them in the LibreEHR database like this "address-line-1 | address-line-2" with a "|" delimiter. When we get the address out of the LibreEHR database and return it through the FHIR API to the iPhone, we need to reassemble the address into two lines (two entries in the lines array) address-line-1 and address-line-2.